content/francais/python-turtle/activity-4.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Activité 4 - Simplifier le code avec des fonctions"
+date: 2024-09-17T13:24:17-07:00
+weight: 6
+draft: false
+---
+
+Nous avons dessiné un hexagone. Mais ce n'est qu'un des nombreux hexagones qu'Alex doit construire pour sa ruche. Encore une fois, il semble que nous devrions répéter le code plusieurs fois pour dessiner plusieurs hexagones. Heureusement, nous n'en avons pas besoin ! Les fonctions à la rescousse !
+
+{{% notice tip %}}
+
+## Fonctions
+
+Une fonction est un moyen de regrouper des lignes de code pour faire quelque chose. Par exemple, `turtle.forward(50)` déplace la tortue de 50 pas en avant, et `turtle.left(120)` tourne la tortue de 120 degrés à gauche. Que faire si nous voulons toujours effectuer ces deux actions ensemble ? Nous pouvons les mettre dans une fonction appelée `draw_line()`, de sorte que chaque fois que cette fonction est appelée, ces deux actions se produisent simultanément. Pensez-y comme une recette : nous regroupons toutes les instructions ensemble, et le nom de la recette est la nourriture que nous préparons.
```

Comment:
Nitpick: is adding an extra space before the colon. 
"Pensez-y comme une recette **:** nous regroupons"

Reply:
updated